README.md

```diff
@@ -1,3 +1,55 @@
 # Argyle iOS SDK example project
 
-Documentation can be found [here](https://argyle.io/docs/argyle-link/ios-sdk-integration)
+Argyle Link iOS SDK provides a way to integrate [Argyle Link](https://argyle.io/docs/argyle-link) into your iOS app.
```

Comment:
This redirects to https://argyle.com/docs/products/argyle-link, so maybe it's worth linking to this directly?

Reply:
All the necessary integration details are now added here, added the url only for context. Do you think it's unnecessary?

Reply:
I think it's good to keep the link in here

Reply:
Misunderstood - I didn't add exact url in purpose, as the content there mirrors what's listed here with the changes.